### PR TITLE
add var authorized_key_exclusive

### PR DESCRIPTION
**Pull request by kastras**
we modify the value of exclusive in module authorized_key for put a variable called authorized_key_exclusive

### CODE REVIEW

**Review by jotasixto — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ ansible-galaxy install -r requirements.yml -p roles/
 
 Role Variables
 --------------
-
+public_keys_path: path for public keys. Default: public_keys
+public_all_filekeys_path: temporary path for work with public keys. Default : /tmp/all_public_keys
+sshlog_enable: for enable sshlog . Default: 0
+authorized_key_exclusive: exclusive public keys of publick_keys_path. Default: yes
```
Comment (alambike) [inline README.md]: extra `k` in  `publick_keys_path`, also I would use backticks to highlight this config options

**Comment by alambike** [comment]
@kastras please make sure to use "Squash & merge" strategy

File: README.md (hunk 1 above)
Comment (kastras) [reply README.md]: Fixed the extra k in public and put the backticks for the new vars in readme file.

**Pull request merged**